lib/formatador/table.rb

```diff
@@ -80,7 +80,11 @@ def display_compact_table(hashes, keys = nil, &block)
   private
 
   def length(value)
-    value.to_s.gsub(PARSE_REGEX, '').length
```

Comment:
Oh, That's better.

Then, Can I use autoload?
I'll remove unicode from dependency, and I'll add `autoload Unicode 'unicode' `to load automatically.